assets/js/components/shapeshift/shift-confirm.component.js

```diff
@@ -19,7 +19,7 @@ function ShiftConfirmController (AngularHelper, $scope, Exchange, Wallet, Ethere
 
   $scope.fee = this.fee;
   $scope.quote = this.quote;
-  $scope.total = parseFloat($scope.quote.depositAmount) + parseFloat($filter('convert')(this.fee, $scope.quote.fromCurrency, false));
+  $scope.total = quote.depositAmount + this.fee;
```

Comment:
When I try to check this I get:
<img width="571" alt="screen shot 2017-08-23 at 7 53 36 am" src="https://user-images.githubusercontent.com/5640772/29614455-327210ca-87d8-11e7-8856-772f45ad22c6.png">